specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -3191,6 +3225,24 @@
       },
       "description": "Application list operation result."
     },
+    "ServicePrincipalObjectResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Service Principal Application Object Id"
```

Comment:
Let's use the description from the doc:
`The Object ID of the service principal with the specified application ID.`

Reply:
Why it is an array? I thought each app should only have a single service principal in the same tenant. Also, the description is `Gets an object id for a given application id from the current tenant.`, which to me means a singular value

Reply:
Service json format is array. Please observe example above.

This was incorporated as a comment resolution. See above @hari comment